xtask/src/xtask.rs

```diff
@@ -181,11 +181,25 @@ fn test_tmt(sh: &Shell) -> Result<()> {
     println!("Discovered plans: {all_plan_files:?}");
 
     cmd!(sh, "cargo run -p tests-integration run-vm prepare-tmt").run()?;
-    // cc https://pagure.io/testcloud/pull-request/174
-    cmd!(sh, "rm -vf /var/tmp/tmt/testcloud/images/disk.qcow2").run()?;
 
     for (_prio, name) in all_plan_files {
-        if let Err(e) = cmd!(sh, "tmt run plans -n {name}").run() {
+        // cc https://pagure.io/testcloud/pull-request/174
+        cmd!(sh, "rm -vf /var/tmp/tmt/testcloud/images/disk.qcow2").run()?;
+        let verbose_enabled = std::env::var("TMT_VERBOSE")
+            .ok()
+            .and_then(|s| s.parse::<u32>().ok())
+            .unwrap_or(0);
+
+        let verbose = if verbose_enabled == 1 {
+            Some("-vvvvv".to_string())
+        } else {
+            None
+        };
```

Comment:
`let verbose = (verbose_enabled == 1).then_some("-vvvvv");` is a bit shorter, also avoids the unnecessary `to_string()` allocation here.